migration/sql-files/029-identities-foreign-key.sql

```diff
@@ -0,0 +1,2 @@
+-- Refactor Identities: add a foreign key constraint
+alter table identities add constraint identies_user_id_users_id_foreign foreign key (user_id) REFERENCES users (id);
```

Comment:
@kwk `ON DELETE SET NULL` seems like the best option, so we could retain the identity as an orphan even if the user had been removed.
@hectorj2f yes,  a review of the foreign keys (existing and missing) would be necessary, indeed.